src/components/BoardColumn/ColumnHeader.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <div class="flex justify-between pb-4">
+  <div
+    class="bg-gradient-to-r from-indigo-500 via- to-indigo-800 text-white font-semibold px-6 py-3 rounded-tr-lg rounded-tl-lg flex justify-between pb-4"
```

Comment:
What is the purpose of the class `via-`?
`py-3` isn't working properly right?

Reply:
I actually took 'py-3' from the tailwind 'gradient'document, will try to remove it if it does not make a difference :)

Reply:
oh and I forgot to delete 'via-'

Reply:
I just checked and if I deleted py-3 it does not look nice as there is no space so left it!